packages/storybook/stories/Localization.stories.mdx

```diff
@@ -8,3 +21,86 @@ Localization is a key to making products that can be used across the world and b
 UI Library will provide out of the box support for some languages and capabilities such as RTL.
 Developers can provide their own localization files to be used for the UI Library.
 These localization capabilities will be added ahead of General Availability.
+
+## LocalizationProvider
+
+`LocalizationProvider` is a wrapping component that sets all the strings for ACS components using a `Locale`. We have a set of
+locales available for import to use. LocalizationProvider is not required as all components default to using our English (US) locale.
+
+### Usage
+
+To use the LocalizationProvider, simply import it from `@azure/communication-react` and wrap it around ACS UI SDK components,
+For the example below we will localize buttons in a `ControlBar` in French (France) and import that locale `fr_FR` from `@azure/communication-react`.
+
+<Source code={LocalizationProviderSnippetText} />
+
+<Canvas withSource="none">
+  <LocalizationProviderSnippet />
+</Canvas>
+
+## Overriding our locales
+
+There are two options to customize the locales that we provide. For the examples below, let's say you wish to have a `ControlBar`
+with strings from our British English locale but you want to change the label of `CameraButton` to "Start" when its state is off.
+
+### Option 1: Import and customize locales
+
+The first option is to import and customize locales. First, import `en_GB` locale from `@azure/communication-react` and then reassign the
+value `strings.cameraButton.offLabel` like shown below.
+
+<Source code={CustomLocaleSnippetText} />
+
+<Canvas withSource="none">
+  <CustomLocaleSnippet />
+</Canvas>
+
+`Locale` stores all strings in the `strings` property. The component name is the first identifer and the string key is the second identifier such
+that each string can be accessed in this pattern `strings.<componentName>.<stringKey>`. If you are using typescript and have intellisense available
+in your IDE, it will be easy to discover all the strings.
+
+### Option 2: Component strings prop
+
+The second option is also easy to pass custom strings directly through the `strings` prop. Each component in ACS has an optional `strings`
+property that interfaces all strings that can be overriden in that component. Like Option 1, import `en_GB` from `@azure/communication-react`.
+Then assign the string directly through `CameraButton` like shown below. This option would not be recommended if a component is used multiple
+times as you would need to assign the `strings` multiple times.
+
+<Source code={StringsPropertySnippetText} />
+
+<Canvas withSource="none">
+  <StringsPropertySnippet />
+</Canvas>
+
+## Managing expansive translations
+
```

Comment:
I am a bit unsure about the value of this section.

* Are we really helping the develoeprs with this section?
* Do we agree that these are good recommendations (especially styling via CSS can get iffy..)

Reply:
Yeah I was unsure about this too. Documentation was what we had in mind initially to address the [user story regarding strings fitting well](https://skype.visualstudio.com/SPOOL/_workitems/edit/2482290). I was hoping to field some feedback on this from my demo. I take it your vote would be against recommendations? Interested in what others think. We can see what customers think and adapt to what they say. I do have something in mind about a truncation prop for `Locale`.

Reply:
Lets just remove this long translation for now. I think if we're offering solutions but they dont fix everyone's scenarios then it will add confusion.